index.html

```diff
@@ -1,3 +1,130 @@
+                    <span>make checkstyle</span>
+                    <p class="details">
+                        Check lint, checkstyle rules. e.g. <a href="https://www.pylint.org/">pylint</a>, <a href="https://github.com/squizlabs/PHP_CodeSniffer">phpcs</a>, <a href="https://eslint.org/">eslint</a>, <a href="https://github.com/checkstyle/checkstyle">checkstyle <em>(java)</em></a> etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make complexity</span>
+                    <p class="details">
+                        Ciclomatic complexity check <em><a href="https://en.wikipedia.org/wiki/Cyclomatic_complexity">(McCabe)</a></em>, <a href="https://pypi.python.org/pypi/radon">radon <em>(python)</em></a>, <a href="https://eslint.org/docs/rules/complexity">eslint (js)</a>, <a href="https://phpmd.org/">PHPMD</a>, <a href="http://www.scalastyle.org/rules-0.5.0.html">rules</a> <em>(scala)</em> etc ...
+                    </p>
+                </li>
+                <li>
+                    <span>make fmt</span>
+                    <p class="details">
+                        Format code. e.g <a href="https://prettier.io/">Prettier <em>(js)</em></a>, <a href="https://golang.org/pkg/fmt/">fmt <em>(golang)</em></a>
+                    </p>
+                </li>
+                <li>
+                    <span>make test</span>
+                    <p class="details">
+                        Shortcut to launch function tests, integration tests and unit tests. <a href="https://docs.pytest.org/en/latest/">pytest</a>, <a href="https://facebook.github.io/jest/">jest (js)</a>, <a href="https://phpunit.de/">phpunit</a>, JUnit <em>(java)</em> etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make test-unit</span>
+                </li>
+                <li>
+                    <span>make test-functional</span>
+                </li>
+                <li>
+                    <span>make test-integration</span>
+                </li>
+                <li>
+                    <span>make run</span>
+                    <p class="details">
+                        Run locally the application, e.g. node index.js, python -m myapp, go run mypapp etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make watch</span>
+                    <p class="details">
+                        Hot reloading for development use. 
+                    </p>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section>
+        <h2>Story</h2>
+        <p>
+            Developers live in a technological world which is continuously changing.<br />
+            Projects switching, technologies switching, languages switching, a big bunch of switching, don't it ?<br />
+            At dailymotion, to take care of the mental health of our engineers, as they are working over 500+ repositories,
```

Comment:
At dailymotion, we take care of the mental health of our engineers, as they work on over 500+ repositories,
developed with more than 50 technological stacks. Therefore we decided to re-think our way to deal with the complexity of our internal codebase.